lib/process-release.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable node/no-deprecated-api */
```

Comment:
`url.resolve()` and `url.parse()` are used in here. I'm not entirely comfortable switching over fully to `URL` yet, it took a little while to stabilise, so maybe not until we want to drop support for Node 8, or until Node starts printing deprecation warnings to stdout.